packages/pre/src/porter.ts

```diff
@@ -0,0 +1,17 @@
+import { getPorterUri as doGetPorterUri } from '@nucypher/shared';
+
+type PreDomain = 'oryx';
+
+export const PRE_DOMAIN: Record<string, PreDomain> = {
+  TESTNET: 'oryx',
```

Comment:
Was this supposed to be `tapir`?

```suggestion
  TESTNET: 'tapir',
```

Reply:
Support `tapir`, `mainnet` and `lynx` networks. Test examples and demos against `lynx`.

Reply:
@derekpierre What was the conclusion here? To support `tapir`, `mainnet`, and `lynx` networks in both PRE and TACo?

Reply:
Yes. All possible TACo domains should be allowed: `tapir`, `lynx`, `mainnet`.